test/k8s/updates.go

```diff
@@ -1,536 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-// Copyright Authors of Cilium
-
-package k8sTest
-
-import (
-	"fmt"
-	"path/filepath"
-	"strconv"
-	"strings"
-	"time"
-
-	. "github.com/onsi/gomega"
-
-	"github.com/cilium/cilium/pkg/versioncheck"
-	. "github.com/cilium/cilium/test/ginkgo-ext"
-	"github.com/cilium/cilium/test/helpers"
-)
-
-const (
-	// cluster name and ID are set to provide coverage for changes to
-	// ClusterMesh, including updates to BPF map updates
-	clusterName = "test-upgrade-downgrade"
-	// clusterID of 170(10101010) was chosen to utilize all 8 bits of the
-	// default ClusterID size.
-	// note: It's possible to extend the size of ClusterID by setting
-	// MaxConnectedClusters, however this feature is only available to new
-	// clusters - thus upgrade and downgrade are not possible. Test coverage
-	// for this feature should be added after it is released.
-	clusterID = "170"
-)
-
-var (
-	// These are set in BeforeAll
-	demoPath         string
-	l7Policy         string
-	migrateSVCClient string
-	migrateSVCServer string
-	stableChartPath  string
-)
-
-var _ = Describe("K8sUpdates", func() {
-
-	// This test runs 8 steps as following:
-	// 1 - delete all pods. Clean cilium, this can be, and should be achieved by
-	// `clean-cilium-state: "true"` option that we have in configmap
-	// 2 - install cilium `cilium:v${LATEST_STABLE}`
-	// 3 - make endpoints talk with each other with policy
-	// 4 - upgrade cilium to `k8s1:5000/cilium/cilium-dev:latest`
-	// 5 - make endpoints talk with each other with policy
-	// 6 - downgrade cilium to `cilium:v${LATEST_STABLE}`
-	// 7 - make endpoints talk with each other with policy
-	// 8 - delete all pods. Clean cilium, this can be, and should be achieved by
-	// `clean-cilium-state: "true"` option that we have in configmap.
-	// This makes sure the upgrade tests won't affect any other test
-	// 9 - re install cilium:latest image for remaining tests.
-
-	var (
-		kubectl *helpers.Kubectl
-
-		cleanupCallback = func() {}
-	)
-	BeforeAll(func() {
-		SkipIfIntegration(helpers.CIIntegrationAKS)
-		canRun, err := helpers.CanRunK8sVersion(helpers.CiliumStableVersion, helpers.GetCurrentK8SEnv())
-		ExpectWithOffset(1, err).To(BeNil(), "Unable to get k8s constraints for %s", helpers.CiliumStableVersion)
-		if !canRun {
-			Skip(fmt.Sprintf(
-				"Cilium %q is not supported in K8s %q. Skipping upgrade/downgrade tests.",
-				helpers.CiliumStableVersion, helpers.GetCurrentK8SEnv()))
-			return
-		}
-
-		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
-
-		demoPath = helpers.ManifestGet(kubectl.BasePath(), "demo.yaml")
-		l7Policy = helpers.ManifestGet(kubectl.BasePath(), "l7-policy.yaml")
-		migrateSVCClient = helpers.ManifestGet(kubectl.BasePath(), "migrate-svc-client.yaml")
-		migrateSVCServer = helpers.ManifestGet(kubectl.BasePath(), "migrate-svc-server.yaml")
```

Comment:
These manifests look unused now. Delete them as well?

Reply:
I saw folks referencing them in other repos :grimacing:

Reply:
I will keep this for now until we have resolved the external reference.

Reply:
In principle they could still reference the manifests from a stable branch, e.g. v1.14.